app/src/chainlit.py

```diff
@@ -40,7 +40,7 @@ def engine_url_query_value() -> str:
     # Using this suggestion: https://github.com/Chainlit/chainlit/issues/144#issuecomment-2227543547
     parsed_url = urlparse(url)
     qs = parse_qs(parsed_url.query)
-    return qs.get("engine", [os.environ.get("CHAT_ENGINE", "default_engine")])[0]
+    return qs.get("engine", [os.environ.get("CHAT_ENGINE", "guru-snap")])[0]
```

Comment:
Sorry I missed this in an earlier review -- we should set this in AppConfig and put a default value there, then use AppConfig().chat_engine to retrieve the configuration value (consistent with how the rest of the app is configured)_. AppConfig uses Pydantic to auto-pull environment variables so it can still be overridden by setting CHAT_ENGINE in the environment if needed.